### ISSUE TITLE
More detailed breakdown of which teams/users are missing

### ISSUE THREAD

**Issue by joao-paulo-parity** [paritytech/pr-custom-review#7]
The current implementation [shows the `total/needed`](https://github.com/paritytech/pr-custom-review/blob/d7ec668efb47f38cb46ec42600d434b0779b42d6/src/main.ts#L274) but that is not detailed enough. It should instead say e.g.

> - Missing approval from teams: team1, team2
> - Missing aproval from users: user1, user2

However, it's not possible to jam that much text in the [status' description](https://docs.github.com/en/rest/reference/commits#create-a-commit-status) which has a character count limit and is not easily discoverable or readable. The action should instead post a link or a comment with a more detailed breakdown telling which approvals are missing exactly.

**Comment by joao-paulo-parity** [comment]
The explanation is more detailed in #27, as shown in https://github.com/tripleightech/pr-custom-review-test/runs/4804709393?check_suite_focus=true#step:3:14. Do we need to post it as a pull request comment or is it fine for it to stay there?

In #27 I implemented some logic which links directly to the step's output, therefore it'll be easy for people to check out who's missing. Nevertheless that is still different from (perhaps worse than) the approach of posting pull request comments (e.g. https://www.fullstory.com/blog/taming-github-codeowners-with-bots) but I am not sure if we want that. What do you think @TriplEight @sergejparity?

![image](https://user-images.githubusercontent.com/77391175/149356124-91a152bf-3c75-461e-be56-636f53aa3764.png)

**Comment by TriplEight** [comment]
This should not have us blocked from deploying the MVP.

In my view, it's okay to leave it this way, but better add `Error:` before the [message](https://github.com/tripleightech/pr-custom-review-test/runs/4805815620?check_suite_focus=true#step:3:16) and maybe highlight the line in some colour. UX-wise it's a one click more, but we can put as much info as we want there.

In the case of the comment, I think this is more complicated and would require a feature of having only one comment of this bot in one PR. Otherwise, it will be a mess of automated chatter everywhere.

To summarise, let's move on with the basic log output and have comments as an improvement.
